src/scaler/daemonscaler/provider.rs

```diff
@@ -258,6 +281,17 @@ impl<S: ReadStore + Send + Sync> ProviderDaemonScaler<S> {
     }
 }
 
+fn compute_provider_config_hash(provider_config: &CapabilityConfig) -> Option<u64> {
+    match provider_config.try_base64_encoding() {
+        Ok(base64) => {
+            let mut hasher = std::collections::hash_map::DefaultHasher::new();
```

Comment:
I think we just have the hash be a string rather than hashing again into a number. So instead of this function, you can just do `provider_config.try_base64_encoding().ok()` instead

Reply:
cool cool